mvnw.cmd

```diff
@@ -0,0 +1,188 @@
+@REM ----------------------------------------------------------------------------
```

Comment:
@olena-syrotenko this file also should be ignored

Reply:
done